models/books.js

```diff
@@ -0,0 +1,17 @@
+const mongoose = require('mongoose');
+require('dotenv').config();
+
+mongoose.connect(process.env.DATABASE_URL);
+
+const { Schema } = mongoose;
+
+const bookSchema = new Schema({
+    title: String,
+    description: String,
+    status: Boolean,
+    email: String,
+});
+
+const Book = mongoose.model('Book', bookSchema);
+
+module.exports = Book;
```

Comment:
Nice work so far!

Feedback here: Consider naming file `bookSchema` or something similar so that is easy to tell right off the bat what this file contains 👍